utils/zsh/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=zsh
-PKG_VERSION:=5.2
+PKG_VERSION:=5.4.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@SF/zsh
-PKG_MD5SUM:=afe96fde08b70e23c1cab1ca7a68fb34
+PKG_MD5SUM:=0b80b7f64c30397cd747d97c378018af
```

Comment:
okay, got it. Although, It'd be nice to use SHA512 there, but not 256 (since it has collisions too), but anyway, I'll push fix soon.